### PR TITLE
Fix more json errors

### LINKED ISSUE #64707: (65407fa) (Boiled) Makeshift Bandage crafting not creating batches properly.
### Describe the bug

(Prob an issue from Innawoods as well.)

Crafting Makeshift Bandages with the 3x unit or 1x batch with (Fibercloth) Sheets vs Patches isn't working properly.

Patch AND Sheet recipe crafts 1x instead of 1x/8x as prior therefore player must cut up Sheet > Patches to get proper recipe returns.

### Attach save file

[InnaMega Techworld-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/11116587/InnaMega.Techworld-trimmed.tar.gz)

### Steps to reproduce

- Craft 2x Fibercloth Sheet. 
- Disassemble 1x into 8x Patches.
- Craft Makeshift Bandage using Patch and Sheet recipe.
- Both recipes result in 1x Bandage only.

### Expected behavior

Not.

### Screenshots

_No response_

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19044.2728 (21H2)
- Game Version: 65407fa [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Innawood [innawood],
    sees-player icon, HitButton_iso [sees_player_hitbutton],
    sees-player, run, crouch, bleed icons, retrodays [sees_player_retro],
    Translate Complex Dialogue [translate_dialogue],
    Megafauna [megafauna],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    SpeedyDex [speedydex],
    Stats Through Kills [stats_through_kills],
    Stats Through Skills [StatsThroughSkills],
    Tamable Wildlife [Tamable_Wildlife],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Bionic Professions [package_bionic_professions],
    DinoMod [DinoMod]
]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by anothersimulacrum**
#### Summary
None

#### Purpose of change
Fix more JSON errors (mostly in mods).
Follows https://github.com/CleverRaven/Cataclysm-DDA/pull/64494
Fixes https://github.com/CleverRaven/Cataclysm-DDA/issues/64707

#### Describe the solution
Enable unvisited JSON member check, fix what shows up.
See commits for individual fixes.

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
CC @Light-Wave

File: data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_inventor.json [hunk 1]
```diff
@@ -127,7 +127,7 @@
     "material": [ "forged_dreamstuff" ],
     "symbol": "?",
     "looks_like": "textbook_anarch",
-    "scannable": false,
+    "//scannable": [ false, "not a book, but shouldn't be scannable" ],
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_inventor.json:130]: Is this how making something not scannable should work or does the not scannable object not exist?

Comment (anothersimulacrum) [reply data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_inventor.json:130]: These are not books, so they can't be scanned. I'm just keeping this as a comment should that change in the future.

**Review by Light-Wave — APPROVED** [review]

File: data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_classless.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "material": [ "forged_dreamstuff" ],
     "symbol": "0",
     "color": "green",
-    "scannable": false,
+    "//scannable": [ false, "not a book, but shouldn't be scannable" ],
```
Comment (GuardianDll) [inline data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_classless.json:15]: isn't `scannable` was introduced in #63508? maybe something wrong with this pr?

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 3]
```diff
@@ -1335,7 +1289,6 @@
     "material": [ "wood" ],
     "symbol": "!",
     "color": "green",
-    "stack_size": 10,
```
[the remark this answers is not in the document]
Comment (anothersimulacrum) [reply data/mods/Magiclysm/items/ethereal_items.json:1338]: 60ml results in this item failing the density test. https://github.com/CleverRaven/Cataclysm-DDA/actions/runs/4596784422/jobs/8118689397?pr=64800#step:16:1544

**Pull request merged**